ProjFS.Mac/PrjFSKext/PrjFSKext/KauthHandler.cpp

```diff
@@ -575,7 +575,7 @@ static int HandleFileOpOperation(
 
 static bool ShouldHandleVnodeOpEvent(
     // In params:
-    vfs_context_t context,
+    vfs_context_t __nonnull context,
```

Comment:
Why do we have `__nonnull` (two underscores, lower case) here and `_Nonnull` (one underscore, upper case N) everywhere else? Do they mean anything different?

Reply:
Ah, I missed that one. I originally started putting in the `__nonnull`/`__nullable` in, then remembered that that's the old syntax and [you're supposed to use `_Nonnull` etc. now](https://clang.llvm.org/docs/AttributeReference.html#nullability-attributes) Looks like I missed one in the find-all.